test/e2e/_helpers.js

```diff
@@ -14,8 +15,30 @@ import personService from '../../common/services/person'
 import personEscortRecordService from '../../common/services/person-escort-record'
 import profileService from '../../common/services/profile'
 import referenceDataService from '../../common/services/reference-data'
+import { SENTRY } from '../../config'
 import { formatDate } from '../../config/nunjucks/filters'
 
+if (SENTRY.DSN) {
+  Sentry.init({
+    dsn: SENTRY.DSN,
+    environment: SENTRY.ENVIRONMENT,
+  })
+}
+
+function errorHandler(body) {
+  return err => {
+    Sentry.withScope(scope => {
+      err.errors.forEach((apiErr, idx) => {
+        scope.setContext(`error_${idx}`, apiErr)
+      })
+      scope.setContext('body', body)
```

Comment:
This does send the body data to Sentry but in our cases it is fake data generated using faker so won't contain anything sensitive/PII.